portal-web/docroot/html/taglib/ui/search_iterator/list.jsp

```diff
@@ -320,11 +320,17 @@ if (fixedHeader) {
 						%>
 
 							<td class="<%= columnClassName %>" colspan="<%= entry.getColspan() %>">
+								<c:if test="<%= !Validator.isBlank(searchContainerRowAriaLabel) %>">
+									<div class="item-selector-cell-item">
```

Comment:
Hey @pat270 why do we still have this here? Shouldn't this have been removed?

Reply:
@ethib137 I missed it. I'll remove it.